src/App.tsx

```diff
@@ -5,6 +5,7 @@ import {Curiosity} from "./Pages/CuriosityLandingPage/CuriosityLandingPage";
 import {Spirit} from "./Pages/SpiritLandingPage/SpiritLandingPage";
 import {Opportunity} from "./Pages/OpportunityLandingPage/OpportunityLandingPage";
 import {NavBar} from "./NavBar/NavBar";
+import {DateInputForm} from "./Components/DateInputForm/DateInputForm";
```

Comment:
is this being used? It doesn't look like it is?

Reply:
It will be used on the "On this Day" page in the Curiosity section with the rover photo api, we hope